pkg/handler/pullrequest/handler.go

```diff
@@ -63,17 +68,12 @@ func logPullRequestInfo(pr *github.PullRequestPayload) {
 	klog.Infof("            name: %s", pr.PullRequest.Base.Ref)
 }
 
-func openOrSync(gitRepo *git.Git, pr *github.PullRequestPayload, gh ghclient.GH) error {
+func openOrSync(gitRepo *git.Git, pr *github.PullRequestPayload, gh ghclient.GH, config *repoconfig.BotConfig) error {
 	prNum := int(pr.Number)
 
-	config, err := repoconfig.Read(pr.PullRequest.Base.Repo.SSHURL, pr.PullRequest.Base.Repo.FullName, pr.PullRequest.Base.Sha)
-	if err != nil {
-		klog.Infof("Error reading bot config: %s", err)
-	}
-
 	readyToReviewMsg := ""
 	if config != nil && config.LabelApproved != nil {
-		readyToReviewMsg += fmt.Sprintf("\n🚀 Full E2E won't run until the %q label is applied. " +
+		readyToReviewMsg += fmt.Sprintf("\n🚀 Full E2E won't run until the %q label is applied. "+
```

Comment:
oh, that's an artifact of my auto gofmt running ;) let's go with it anyway if that's fine.